admin-portal/src/pages/home/components/table/components/tableRow/tableRow.tsx

```diff
@@ -23,6 +24,7 @@ export default function TableRow(props: TableRowProps) {
 
   return (
     <tr key={aidPackage.packageID}>
+      <td>{moment.unix(aidPackage.dateTime).format("DD/MM/YYYY")}</td>
```

Comment:
from what I've tried out, .local() is needed to show the dateTime local to the user
`moment.unix(aidPackage.dateTime).local().format("YYYY-MM-DD hh:mm A")`